app/page.js

```diff
@@ -21,17 +22,24 @@ export default function Page() {
           <h1 className="font-bold text-center text-3xl text-gray-900 ml-1 mt-12">
             Gilgorri Santiago
           </h1>
-          <h2 className="font-bold -text-center text-2xl text-gray-800 ml-6 mt-3">
+          <h2 className="font-bold -text-center text-2xl text-gray-800 mt-3">
             Santiagogilgorri@gmail.com
           </h2>
-          <div className="my-5 m-40">
+          <div className="flex justify-evenly space-x-40">
```

Comment:
Agregale un toque de margen:
```suggestion
          <div className="flex justify-evenly space-x-40 mb-2">
```